.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,8 +18,9 @@ Here's some guidelines to help the review process go smoothly.
    noted here: https://help.github.com/articles/closing-issues-using-keywords/
 
 3. If your pull request is not ready for review but you want to make use of the
-   continuous integration testing facilities please label it with `[WIP]` and
-   then remove the label when you'd like it to be reviewed.
+   continuous integration testing facilities please click the arrow besides
+   "Create Pull Request" and choose "Create Draft Pull Request". And
```

Comment:
I am not sure it is good style to begin a sentence with `And`, you can probably drop that?

Reply:
resolved in commit 0288022